README.md

```diff
@@ -309,8 +309,19 @@ See template example below for how to continuously update `ids`
 
 #### Automation example
 ```
-      payload: '{{''{"pGrid": ''}}{{states(''sensor.power_use'')}}{{'', "pPv": ''}}{{states(''sensor.solar_production'')}}{{'',
-        "pAkku":0}''}}'
+   alias: Go-E Überschussladen
+   description: ""
+   trigger:  
+      - platform: time_pattern    
```

Comment:
Home Assistant recommends indents using 2 spaces (https://developers.home-assistant.io/docs/documenting/yaml-style-guide/). Could you change the indents? Thanks in advance!